docs/validators/upgrades/v4.0-upgrade.md

```diff
@@ -61,8 +63,19 @@ Create a `v4.0.0` directory and copy the upgrade binary (`v4.0.0`) to the direct
 mkdir -p $HOME/.regen/cosmovisor/upgrades/v4.0.0/bin
 cp ./build/regen $HOME/.regen/cosmovisor/upgrades/v4.0.0/bin
 ```
+Ensure the right `regen` binary has been placed in the directory:
+
+```bash
+$HOME/.regen/cosmovisor/upgrades/v4.0.0/bin/regen version
+```
+
+You should see the following:
+
+```bash
+v4.0.0
```

Comment:
```suggestion
v4.0.1
```

Reply:
Maybe you're still working on the changes but `v4.0.0` is still referenced throughout on this page.